worksheet_B/Mandelbrot.cpp

```diff
@@ -2,6 +2,125 @@
 //
 
 #include "stdafx.h"
+// source for hsv-rgb "http://stackoverflow.com/questions/3018313/algorithm-to-convert-rgb-to-hsv-and-hsv-to-rgb-in-range-0-255-for-both"
+typedef struct {
+	double r;       // percent
+	double g;       // percent
+	double b;       // percent
+} rgb;
```

Comment:
Not your code I know, but `typedef struct` is a hangover from C -- in C++ it's more common to write this as `struct rgb {...};`